renku_theme/login/login.ftl

```diff
@@ -2,6 +2,11 @@
 <@layout.registrationLayout displayMessage=!messagesPerField.existsError('username','password') displayInfo=realm.password && realm.registrationAllowed && !registrationDisabled??; section>
     <#if section = "header">
         ${msg("loginAccountTitle")}
+        <div id="renku-login-terms-container">
```

Comment:
non-blocking: Would it be possible to make this configurable?

Reply:
In theory yes, but the ids are always specified statically in the keycloak templates. If we wanted to make this configurable, then we should switch to using a class to specify the styling, since classes are usually configurable in the keycloak template.

Reply:
I meant: is doable to make have a configuration bit which makes this container appear or not?